packages/target-tools/delivery-api-client/index.js

```diff
@@ -1888,6 +1888,7 @@ class DeliveryApi extends BaseAPI {
     }
     async execute(imsOrgId, sessionId, deliveryRequest, version) {
         const response = await this.executeRaw({ imsOrgId: imsOrgId, sessionId: sessionId, deliveryRequest: deliveryRequest, version: version });
+        

```

Comment:
Let's remove this file from the PR since it's generated and there's only leftover whitespace